tendrl/commons/flows/import_cluster/__init__.py

```diff
@@ -86,6 +86,20 @@ def run(self):
             if is_mon:
                 import_ceph(NS.tendrl_context.integration_id)
         else:
+            # Check if minimum required version of underlying gluster
+            # cluster met. If not fail the import task
+            sds_version = NS.etcd_orm.client.read(
+                "nodes/%s/DetectedCluster/sds_pkg_version" %
+                NS.node_context.node_id
+            ).value
+            maj_ver, min_ver, rel = sds_version.split('.')
+            req_maj_ver, req_min_ver, req_rel = \
+                NS.config.data['min_reqd_gluster_ver'].split('.')
```

Comment:
Ok. Will add one data point here. Can you guide me to the section where this should be added.